static/templates/settings/playground_settings_admin.hbs

```diff
@@ -5,6 +5,43 @@
             Configure external sites that can run code present in a Zulip code block.
```

Comment:
I would change this to: "Configure external sites that can run code from a Zulip code block."

Reply:
We do link to the Help Center article from the blue question-mark symbol on the page.

Reply:
Ah, good point! I guess whether or not we want to mention the Help Center in the text may depend on whether it's helpful for understanding the overall context vs. just instructions for how to add a new playground (what I'd expect to see under `?`).